File: src/my_robot_tutorials/src/radio_transmitter.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char **argv) {
    ros::init(argc, argv, "radio_transmitter", ros::InitOption::AnonymousName);
    auto node_handle = ros::NodeHandle();
    auto publisher = node_handle.advertise<std_msgs::String>("/robot_news_radio", 10);
    auto rate = ros::Rate(10);

    ROS_INFO("Node has been started");

    while (ros::ok()) {
        auto msg = std_msgs::String();
        msg.data = "[C++] Hi, this is radio_transmitter node speaking!";
        publisher.publish(msg);
        rate.sleep();
    }
}

File: src/my_robot_tutorials/scripts/radio_transmitter.py
#!/usr/bin/env python3

import rospy
from std_msgs.msg import String


def main():
    rospy.init_node("radio_transmitter", anonymous=True)
    publisher = rospy.Publisher("/robot_news_radio", String, queue_size=10)
    rate = rospy.Rate(10)

    rospy.loginfo("Node has been started")

    while not rospy.is_shutdown():
        msg = String()
        msg.data = "Hi, this is radio_transmitter node speaking!"
        publisher.publish(msg)
        rate.sleep()


if __name__ == "__main__":
    main()
